api/services/pdf_auditor.py

```
@@ -0,0 +1,861 @@
+
+            self.logger.info(f"Auditoria concluída: {filename} - Risk Level: {audit_summary.risk_level}")
+            return report
+
+        except Exception as e:
+            self.logger.error(f"Erro na auditoria de {filename}: {str(e)}")
+            raise
+
+    def _calculate_hashes(self, pdf_bytes: bytes) -> HashInfo:
+        """Calcula múltiplos hashes do arquivo"""
+        return HashInfo(
+            md5=hashlib.md5(pdf_bytes).hexdigest(),
```

Ruff (0.14.4)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)